docker/development-easy-arm/docker-compose.yml

```diff
@@ -0,0 +1,114 @@
+# Use admin/pass as user/password credentials to login to openemr (from OE_USER and OE_PASS below)
+# MYSQL_HOST and MYSQL_ROOT_PASS are required for openemr
+# EASY_DEV_MODE prevents issues with permissions when mounting volumes
+# EAST_DEV_MODE_NEW with value of 'yes' expands EASY_DEV_MODE by not requiring downloading
+#  code from github (uses local repo).
+# MYSQL_USER, MYSQL_PASS, OE_USER, MYSQL_PASS are optional for openemr and
+#   if not provided, then default to openemr, openemr, admin, and pass respectively.
+version: '3.1'
+services:
+  mysql:
+    restart: always
+    image: jsurf/rpi-mariadb
+    command: ['mysqld','--character-set-server=utf8mb4','--ssl-ca=/etc/ssl/ca.pem','--ssl_cert=/etc/ssl/server-cert.pem','--ssl_key=/etc/ssl/server-key.pem']
+    ports:
+    - 8320:3306
+    volumes:
+    - ../development-insane/sql-ssl-certs-keys/easy/ca.pem:/etc/ssl/ca.pem:ro
+    - ../development-insane/sql-ssl-certs-keys/easy/server-cert.pem:/etc/ssl/server-cert.pem:ro
+    - ../development-insane/sql-ssl-certs-keys/easy/server-key.pem:/etc/ssl/server-key.pem:ro
+    - databasevolume:/var/lib/mysql
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+  openemr:
+    restart: always
+    # use an image if you're not changing the build steps:
+    image: openemr/openemr:flex
+    # if editing the Dockerfile, clone the devops repo and point to the path:
+    # build: ../openemr-devops/docker/openemr/flex-edge/
+    ports:
+    - 8300:80
+    - 9300:443
+    volumes:
+    - ../..:/openemr:ro
+    - ../..:/var/www/localhost/htdocs/openemr:rw
+    - publicvolume:/var/www/localhost/htdocs/openemr/public:rw
+    - sitesvolume:/var/www/localhost/htdocs/openemr/sites:rw
+    - nodemodules:/var/www/localhost/htdocs/openemr/node_modules:rw
+    - vendordir:/var/www/localhost/htdocs/openemr/vendor:rw
+    - ccdamodule:/var/www/localhost/htdocs/openemr/ccdaservice:rw
+    - logvolume:/var/log
+    - couchdbvolume:/couchdb/data
+    environment:
+      MYSQL_HOST: mysql
+      MYSQL_ROOT_PASS: root
+      MYSQL_USER: openemr
+      MYSQL_PASS: openemr
+      OE_USER: admin
+      OE_PASS: pass
+      EASY_DEV_MODE: "yes"
+      EASY_DEV_MODE_NEW: "yes"
+      # e2e requires chromium, which alpine isn't giving us in arm
+      # TODO, need to add more granular setting for this to allow other composer developer dependencies, but not e2e
```

Comment:
TODO for me

Reply:
@jesdynf ,
Won't be able to test this until my pi arrives in mail. Does the build completely break since chromium package does not exist?

Reply:
Ideally plan to keep the same setting/process, but ensure script does not break if chromium package does not exist.

Reply:
Full failure, missing chromium package in Alpine means build halts before autoconfiguration.

Reply:
thanks. I'll fix this up when got pi to test it. For now will just keep this switch to turned off.